rpc/src/test.rs

```diff
@@ -544,6 +548,4 @@ fn test_rpc() {
             pretty_assert_eq!(actual, expected, "Assert results of jsonrpc",);
         }
     }
-
-    server.close();
```

Comment:
If we don't close the server explicitly, the test may panic randomly when dropping objects.

Reply:
I think this issue has been fixed by https://github.com/paritytech/jsonrpc/pull/495